presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricKey.java

```diff
@@ -0,0 +1,110 @@
+    public static final RuntimeMetricKey OPTIMIZED_WITH_MATERIALIZED_VIEW_COUNT = new RuntimeMetricKey("optimizedWithMaterializedViewCount");
+    public static final RuntimeMetricKey MANY_PARTITIONS_MISSING_IN_MATERIALIZED_VIEW_COUNT = new RuntimeMetricKey("manyPartitionsMissingInMaterializedViewCount");
+    public static final RuntimeMetricKey SKIP_READING_FROM_MATERIALIZED_VIEW_COUNT = new RuntimeMetricKey("skipReadingFromMaterializedViewCount");
+    public static final RuntimeMetricKey FRAGMENT_RESULT_CACHE_HIT = new RuntimeMetricKey("fragmentResultCacheHitCount");
+    public static final RuntimeMetricKey FRAGMENT_RESULT_CACHE_MISS = new RuntimeMetricKey("fragmentResultCacheMissCount");
+    public static final RuntimeMetricKey GET_VIEW_TIME_NANOS = new RuntimeMetricKey("getViewTimeNanos");
+    public static final RuntimeMetricKey GET_MATERIALIZED_VIEW_TIME_NANOS = new RuntimeMetricKey("getMaterializedViewTimeNanos");
+    public static final RuntimeMetricKey GET_MATERIALIZED_VIEW_STATUS_TIME_NANOS = new RuntimeMetricKey("getMaterializedViewStatusTimeNanos");
+    public static final RuntimeMetricKey GET_TABLE_HANDLE_TIME_NANOS = new RuntimeMetricKey("getTableHandleTimeNanos");
+    public static final RuntimeMetricKey GET_TABLE_METADATA_TIME_NANOS = new RuntimeMetricKey("getTableMetadataTimeNanos");
+    public static final RuntimeMetricKey GET_SPLITS_TIME_NANOS = new RuntimeMetricKey("getSplitsTimeNanos");
+    public static final RuntimeMetricKey LOGICAL_PLANNER_TIME_NANOS = new RuntimeMetricKey("logicalPlannerTimeNanos");
+    public static final RuntimeMetricKey FRAGMENT_PLAN_TIME_NANOS = new RuntimeMetricKey("fragmentPlanTimeNanos");
+    public static final RuntimeMetricKey GET_LAYOUT_TIME_NANOS = new RuntimeMetricKey("getLayoutTimeNanos");
+    public static final RuntimeMetricKey REWRITE_AGGREGATION_IF_TO_FILTER_APPLIED = new RuntimeMetricKey("rewriteAggregationIfToFilterApplied");
+    // Time between task creation and start.
+    public static final RuntimeMetricKey TASK_QUEUED_TIME_NANOS = new RuntimeMetricKey("taskQueuedTimeNanos");
+    // Total operation time of a task on a worker. TASK_ELAPSED_TIME_NANOS - TASK_SCHEDULED_TIME_NANOS is the time when the task is doing nothing, e.g. it might be waiting for splits/inputs.
+    public static final RuntimeMetricKey TASK_SCHEDULED_TIME_NANOS = new RuntimeMetricKey("taskScheduledTimeNanos");
+    // Blocked time for the operators due to waiting for inputs.
+    public static final RuntimeMetricKey TASK_BLOCKED_TIME_NANOS = new RuntimeMetricKey("taskBlockedTimeNanos");
+    // Time taken for a read call to storage
+    public static final RuntimeMetricKey STORAGE_READ_TIME_NANOS = new RuntimeMetricKey("storageReadTimeNanos");
+    // Size of the data retrieved by read call to storage
+    public static final RuntimeMetricKey STORAGE_READ_DATA_BYTES = new RuntimeMetricKey("storageReadDataBytes");
```

Comment:
Why are we creating a RuntimeMetricKey with NONE as RunTimeUnit ? We know its BYTES right ?

Reply:
Another PR for setting the metrics is planned.